### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by IBTechRaj**
I have finished this solo project and submitting for code review.

### CODE REVIEW

**Comment by IBTechRaj** [comment]
This project is completed. Kindly do the code review and suggest improvements. Thanks

**Comment by azdrenymeri** [comment]
### Hello @IBTechRaj  👋 
* The tabs are all the way to the left and are not properly shown.
* Also  the background image has a low quality I would suggest checking [Unsplash](https://unsplash.com/s/photos/restaurant) for free high-quality pictures 
scr 
- [ ] Kindly fix all stickler failed checks.
- [ ]  file `src/all.js` is never used.
### Kindly fix these issues and add it for another review

**Comment by IBTechRaj** [comment]
Carried out the suggestions. Stickler error 2 space indent required, could not be set right. Even though I set 2 spaces for tab and even after manually setting 2 space indent also, Vscode is automatically making indent size to 4. Pl advise me how I can do this.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
STATUS: Changes Required 

Hi, I see your progress everything works fine but we still need stickler to work especially since this is not a big issue. 
Since the plugin/extension to use for indentation varies, lets do this the manual way
 
firstly, go to settings and make sure `editor.autoIndent` is not set

- Look at the bottom right of VScode you would see a tab called spaces, click on it.
- from the dropdown pop up, select 'Indent using Spaces' 
- select 2
- then highlight the code manually and move them backward using 'ctrl + {' (on PC, please confirm the mac shorcut if that is what you use.

 This should work fine now 
Please try to fix it and request for a re-review, also feel free to reach out if this doesn't help :smile: 

Raphael Noriode :sunglasses:

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @IBTechRaj  👋 
#### The background image looks cooler 👍 
But still, there is this error:
scr
* Also add `.vscode` on your `.gitignore` so it doesn't get published on your remote repository.

**Comment by IBTechRaj** [comment]
fixed the errors and made suggested changes. Kindly review the code. Thanks

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @IBTechRaj  :clap: 

**Great work fixing the error message from the previous review.**
**Just little bug to work on**
when i clicked on `about`,`menu` and `contact` nothing changes on the page, instead of being refer to the menu page, menu page or contact page but for some bug reasons the page remains the same after clicking the three `nav` bar.

image


**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**


Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: eslint.config [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    'eol-last': 0,
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  {
+    "no-use-before-define": ["error", { "functions": false, "classes": false }]
+}
+}
```
Comment (bolah2009) [inline eslint.config:16]: [NEXT PROJECT]

Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/javascript).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good job implementing the suggested changes, please not the following especially for next projects:
- Use the correct linter config, see comment below.
- Add a descriptive README to your projects

   > * [ ] The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- Add a descriptive PR Description.

   > * [ ] Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Good luck with your next projects. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

**Pull request closed without merging**